freqtrade/optimize/backtesting.py

```diff
@@ -162,7 +153,7 @@ def backtest(args) -> DataFrame:
     if record and record.find('trades') >= 0:
         logger.info('Dumping backtest results')
         misc.file_dump_json('backtest-result.json', records)
-    labels = ['currency', 'profit_percent', 'profit_BTC', 'duration', 'profit', 'loss']
+    labels = ['currency', 'profit_percent', 'profit_BTC', 'duration']
```

Comment:
Why the drop?

Reply:
No need to get this done here, this is done in the function generating the  table with all results

Reply:
Cool! That would speed up hyperopt as well!